src/implementations/vanilla/components/fact-figures/fact-figures-ec.scss

```diff
@@ -14,20 +14,16 @@
 }
```

Comment:
on the mockups there is no background color on EC

Reply:
Missing on the mockups, designers 'll be updated.